Controllers/tutorials.js

```diff
@@ -38,7 +44,7 @@ const updateSingleTutorial = async (req, res) => {
   try {
     const singleTutorial = await Tutorial.findOneAndUpdate(
```

Comment:
There is also a findByIdAndUpdate method.

Reply:
Didn't change the methods to their "Id", because I fetch the data based on two parameters now, {id, userId}.